README.md

```diff
@@ -23,24 +23,18 @@ It is possible to call `setFocusBehavior()` arbitrarily many times before callin
 ```js
 // Example #1 - unconditional decision.
 
-const controller = new CaptureController;
+const controller = new CaptureController();
 controller.setFocusBehavior("focus-captured-surface");
-const stream = await navigator.mediaDevices.getDisplayMedia({
-  video: true,
-  controller: controller
-});
+const stream = await navigator.mediaDevices.getDisplayMedia({ controller });
```

Comment:
1. I want to be explicit; I think it makes things clearer for a wider audience.
2. The `video: true` fix has still not made it into Chrome's active stable revision. People could stumble on this. I'd rather just include it.

Reply:
1. Does not make sense as the wider audience doesn't need to know a `video` attribute.
2. This makes sense to me. Let's not remove `video: true` then.

Reply:
1. There are two changes here; the change of `controller: controller` into `controller` and removal of `video: true`. My first bullet point refers to the former; my second bullet point refers to the latter.
2. Cool.

Reply:
1. I'd use `controller`. See https://caniuse.com/mdn-javascript_operators_object_initializer_shorthand_property_names

Reply:
1. How sure are we that everyone is aware of this notation?

Reply:
That **everyone** is aware of this notation, zero change.
Since it has been there since 2015 according to https://caniuse.com/mdn-javascript_operators_object_initializer_shorthand_property_names, I'm confident most web developers know this.

Reply:
I said "everyone"? Oops. I meant "most readers."
If you think it's commonplace enough, sure, let's do it. But I think `{video: true, controller}` is awkward. But you're the JS guy, I'm the C++ guy, so...

Reply:
If we're keeping `video: true`, then we shouldn't change your code.